Assets/WebGLTemplates/AirConsole 2020/index.html

```diff
@@ -0,0 +1,119 @@
+﻿<!doctype html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0"/>
+    <style>
+      html, body {
+        font-family: sans-serif;
+        text-align: center;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+      }
+
+      #game-container {
+        position: absolute;
+        left: 50%;
+        top: 0;
+        transform: translate(-50%, 0%);
+      }
+
+      #unity-canvas {
+        height: 100%;
+        width: 100%;
+        background-image: url('TemplateData/loading-screen.png') !important;
+        background-color: black !important;
+        background-position: 50% 0 !important;
+        background-repeat: no-repeat !important;
+        background-size: contain !important;
+      }
+    </style>
+    <script src="https://www.airconsole.com/api/airconsole-1.7.0.js"></script>
+    <script src="Build/{{{ LOADER_FILENAME }}}"></script>
+    <script src="./airconsole-unity-plugin.js"></script>
+    <script src="translation.js"></script>
```

Comment:
<script>
var AIRCONSOLE_TRANSLATION = {{{AIRCONSOLE_TRANSLATION}}};
</script>

Reply:
Can't define custom variables other than in the Player Settings, and they can only be Strings